tools/test_tpm_ek_chain.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+WORKDIR="$(mktemp -d)"
+declare -i MAX_CERT_SIZE=0
+declare -i CHAIN_LEN=0
+declare -i CHAIN_SIZE=0
+
+function cleanup {
+	rm -rf $WORKDIR
+	exit
+}
+
```

Comment:
I know you describe it in the issue, but it does not cost too many lines of code to check for the tools we need and inform user if they are missing. We need to make it as smooth as possible experiecne if we want to hope for some input from community.